.gitmodules

```diff
@@ -55,3 +55,21 @@
 [submodule "third_party/src/github.com/inconshreveable/mousetrap"]
 	path = third_party/src/github.com/inconshreveable/mousetrap
 	url = https://github.com/inconshreveable/mousetrap.git
+[submodule "third_party/glslang"]
```

Comment:
Just a note, shaderc is only guaranteed to work against github.com/google/glslang. 
google/glslang is kept reasonably up-to-date with khronos/glslang, it has not diverged, but we are a few revisions back to make sure that it always works with ToT shaderc.